convert2rhel/pkgmanager/handlers/dnf/__init__.py

```diff
@@ -121,9 +128,9 @@ def _resolve_dependencies(self):
             loggerinst.debug("Got the following exception message: %s" % e)
             loggerinst.critical("Failed to resolve dependencies in the transaction.")
 
-        loggerinst.info("Downloading the packages that were added to the dnf transaction set.")
+        loggerinst.info("\nDownloading the packages that were added to the dnf transaction set.")
```

Comment:
A lot of the log changes prepend a newline, I've not run it myself to check but is it necessarily better?

Reply:
It is easier to follow the output on the terminal. Before, every line was below each other, which, in my opinion, make this things a little harder to see. 

The idea here is to separate by "concerns" the logs, so the depsolver will be together, and the download will be separate from the depsolver, but the logs that follow the packages download will be together too and so on...

Reply:
I think stylistically, logs should generally be line oriented, one line per entry but what you say makes perfect sense for terminal output.  I'm not sure if we can get both behaviours with the way we use logger for displaying the things we want the user to read....  

Maybe we could write some code in our custom logger to strip leading and trailing newlines from the message just when outputting to a log file, but not to the terminal?  Or we could create a set of functions that will output strings to the terminal only (It is a little more than simply calling print() because we would only want it to run if the logging level is appropriate).

Reply:
I think that might be a lot of extra work. So I vote for us reverting this and going with per-line logs rather than empty lines

Reply:
No worries. Will revert the change in the next push.